hooks/post-checkout

```diff
@@ -3,7 +3,19 @@
 set -euo pipefail
 
 EJSON2ENV_PLUGIN_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")"; cd ..; pwd)
-EJSON2ENV_BIN_PATH="$EJSON2ENV_PLUGIN_ROOT/ejson2env"
+arch="$(arch)"
+case "$arch" in
+  arm*|aarch64)
+    arch="arm64";;
```

Comment:
Do we need a `break` in here? I suspect the flow will always fall through if we don't

```suggestion
    arch="arm64"; break;;
```

Reply:
`;;` breaks the case statement.  I've also confirmed it works on both architectures. Docs:

> Each clause must be terminated with ‘;;’, ‘;&’, or ‘;;&’.

https://www.gnu.org/software/bash/manual/html_node/Conditional-Constructs.html

Reply:
No, I must be confusing this with something else.

It works fine in my experimentation.

Reply:
Thanks @stevehodgkiss 👍